go/uidmap/hardcoded_uids.go

```diff
@@ -21,14 +21,14 @@ var usernameSortOrder []uint16
 var usoOnce sync.Once
 
 func findInit() {
-	usoOnce.Do(func() {
+	once.Do(func() {
 		initUsernameOffsets()
 		initUsernames()
 	})
 }
 
 func usernameFindInit() {
-	once.Do(initUsernameSort)
+	usoOnce.Do(initUsernameSort)
```

Comment:
Thanks!!!

Reply:
if we're merging this PR after the release should we pick this change out?

Reply:
No, this is the only thing that uses it.